toxcore/TCP_connection.c

```diff
@@ -1467,7 +1467,7 @@ static void kill_nonused_tcp(TCP_Connections *tcp_c)
     unsigned int i;
     unsigned int num_online = 0;
     unsigned int num_kill = 0;
-    VLA(unsigned int, to_kill, tcp_c->tcp_connections_length);
+    unsigned int* const to_kill = malloc(tcp_c->tcp_connections_length * sizeof(unsigned int));
```

Comment:
`calloc` and move `*`.

Reply:
I use calloc where VLA was followed by zeroing memory via `memset`. In other cases malloc (for performance reasons.